repos/fdbt-site/src/pages/api/additionalPricingStructures.ts

```diff
@@ -0,0 +1,88 @@
+import { NextApiResponse } from 'next';
+import { ADDITIONAL_PRICING_ATTRIBUTE } from '../../../src/constants/attributes';
+import { redirectTo } from '../../../src/utils/apiUtils';
+import { isCurrency } from '../../../src/utils/apiUtils/validator';
+import { updateSessionAttribute } from '../../../src/utils/sessions';
+import { AdditionalPricing, ErrorInfo, NextApiRequestWithSession } from '../../interfaces';
+
+export const checkInputIsValid = (inputtedValue: string | undefined, inputType: string): string => {
+    let error;
+
+    if (!inputtedValue) {
+        error = `Enter a value for the ${inputType}`;
+    } else if (Math.sign(Number(inputtedValue)) === -1) {
+        error =
+            inputType === 'structure discount'
```

Comment:
If we change the inputType as recommended in the below comment, we can just do 

`error = `${inputType}` cannot be a negative number `